pkg/cmd/pulumi/new.go

```diff
@@ -481,6 +497,9 @@ func newNewCmd() *cobra.Command {
 	cmd.PersistentFlags().StringVar(
 		&args.secretsProvider, "secrets-provider", "default", "The type of the provider that should be used to encrypt and "+
 			"decrypt secrets (possible choices: default, passphrase, awskms, azurekeyvault, gcpkms, hashivault)")
+	cmd.PersistentFlags().BoolVarP(
+		&args.listTemplates, "list-templates", "l", false,
+		"Lists templates installed on the user's machine")
```

Comment:
```suggestion
		"List locally installed templates")
```
We should attempt consistency. The `-f` command uses `Forces` instead of `Force`, but the rest of the commands leave it in the imperative form.

Reply:
I want to differentiate that --list-templates flag is a subcommand rather than an adverb like force or dir.

Any ideas around that?